Gopkg.toml

```diff
@@ -112,7 +112,8 @@ ignored = ["github.com/uber/cadence/.gen"]
 
 [[constraint]]
   name = "go.uber.org/cadence"
-  branch = "master"
+  source = "github.com/yycptt/cadence-client"
+  branch = "fix-open-record"
```

Comment:
Use my repo for now for testing. Will change back after client side PR is merged.